basic-algorithms.js

```diff
@@ -1,7 +1,67 @@
 // Names and Input
+var hacker1 = 'Peter Pan';
+
+console.log("The Driver's name is " + hacker1);
+
+var hacker2 = prompt ("What's the navigator's name?");
+
+console.log("The navigator's name is " +hacker2);
 
 
 //Conditionals
+if (hacker1.length > hacker2.length){
+        console.log(`The Driver has the longest name, it has ${hacker1.length} characters`);
+    } else if (hacker1.length == hacker2.length){
```

Comment:
here you should use (hacker1.length === hacker2.length)
to compare the data type AND the value but it's a detail, it's more a good habit to get, 
the exercise is good ;)